tests/boot/test_mcuboot/testcase.yaml

```diff
@@ -7,7 +7,7 @@ common:
       - "I: Starting bootloader"
       - "Launching primary slot application on (.*)"
       - "Secondary application ready for swap, rebooting"
-      - "I: Starting swap using (.*)"
+      - "[I: Starting swap using (.*)|I: Image 0 upgrade secondary slot -> primary slot]"
```

Comment:
wildly incorrect, this means it would match a single character in there, so "I" would pass, "error code 0" would pass, etc.

Reply:
> wildly incorrect, this means it would match a single character in there, so "I" would pass, "error code 0" would pass, etc.

The test application does not print "error code 0".

Reply:
Doesn't matter what it prints, `[` means single character, it is wrong

Reply:
good catch, need remove the '[', let me fix